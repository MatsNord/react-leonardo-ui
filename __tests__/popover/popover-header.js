import React from 'react';
import renderer from 'react-test-renderer';

import Popover from '../../src/popover';

test('Popover header renders correctly', () => {
  const tree = renderer.create(
    <Popover.Header>Header content</Popover.Header>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Popover header with title renders correctly', () => {
  const tree = renderer.create(
    <Popover.Header>
      <Popover.Title>
        Header title
      </Popover.Title>
    </Popover.Header>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Popover header passes props correctly', () => {
  const tree = renderer.create(
    <Popover.Header className="my-class" data-x="x" title="Header title">
      Header title
    </Popover.Header>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
