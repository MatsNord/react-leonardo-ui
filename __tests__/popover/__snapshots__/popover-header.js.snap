// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Popover header passes props correctly 1`] = `
<div
  className="lui-popover__header my-class"
  data-x="x"
  title="Header title"
>
  Header title
</div>
`;

exports[`Popover header renders correctly 1`] = `
<div
  className="lui-popover__header"
>
  Header content
</div>
`;

exports[`Popover header with title renders correctly 1`] = `
<div
  className="lui-popover__header"
>
  <div
    className="lui-popover__title"
  >
    Header title
  </div>
</div>
`;
